### PR TITLE
Add flip for dynamically configuring features

### PR DESCRIPTION
**Pull request by jcoyne**
This will allow us to turn off admin sets for example.

### CODE REVIEW

File: app/controllers/sufia/admin/strategies_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module Sufia
+  module Admin
+    class StrategiesController < Flip::StrategiesController
+      before_action do
+        authorize! :manage, Feature
```
Comment (jeremyf) [inline app/controllers/sufia/admin/strategies_controller.rb]: Why not `Sufia::Feature` as above?

Comment (jcoyne) [inline app/controllers/sufia/admin/strategies_controller.rb]: It is Sufia::Feature since it's being referred to from within the Sufia module.

**Comment by mjgiarlo** [comment]
@jcoyne Did you want your isolate_namespace PR merged in advance, or is this ready?

**Comment by jcoyne** [comment]
@mjgiarlo this is ready. the isolate_namespace will take a bit more work and can come later.

File: app/controllers/sufia/admin/strategies_controller.rb (hunk 1 above)
Comment (mjgiarlo) [inline app/controllers/sufia/admin/strategies_controller.rb]: But aren't both of the new controllers in the `Sufia::Admin` namespace? I don't know why you'd use the explicit `Sufia::` namespace in one and not the other.

Comment (jcoyne) [inline app/controllers/sufia/admin/strategies_controller.rb]: I just wasn't looking at the one when doing the other. The fully-qualified path is optional. Semantically they are the same.

**Pull request merged**